.github/workflows/build.yml

```diff
@@ -0,0 +1,155 @@
+name: Build
+
+on:
+    workflow_call:
+      inputs:
+        platforms:
+          required: true
+          type: string
+        image:
+          required: true
+          type: string
+        tag:
+          required: false
+          type: string
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    permissions:
+      contents: read # for docker/build-push-action to read repo content
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      packages: write # for docker/build-push-action to push to GHCR
+      id-token: write # for docker/login to login to NGINX registry
+    runs-on: ${{ github.event_name != 'pull_request' && contains(inputs.image, 'plus') && 'kic-plus' || 'ubuntu-22.04' }}
```

Comment:
kic-plus?

This logic is confusing me a little bit. What do we run on if not ubuntu?